gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -1209,3 +1212,172 @@ def are_times_in_expected_order(self, times: tuple[datetime.datetime], expected_
             previous_time = instant
         # Return True if no problems found
         return True
+
+    def post_parse_quality_check(self, checks: int = 100, threshold: float = 10e-5):
+        """
+        Master function to check values written after a parse for discrepancies with the source data
+
+        Parameters
+        ----------
+        checks
+            The number of values to check. Defaults to 100.
+        threshold
+            The tolerance for diversions between original and parsed values.
+            Absolute differences between them beyond this limit will raise a ValueError
+        """
+        if self.skip_post_parse_qc:
+            self.info("Skipping post-parse quality check since the source data is different from the parsed data")
```

Comment:
The caller can set skip_post_parse_qc to True for any reason though, so I think either change that behavior to not being configurable or this message should just say something generic like Skipping post-parse QC because skip_post_parse_qc is set